test/apps_test.py

```diff
@@ -0,0 +1,84 @@
+"""Test apps.py module"""
+from unittest import TestCase
+from os.path import join, expanduser
+import pytest
+
+from projector_installer.apps import get_app_path, is_path_to_app, parse_version, \
+    get_data_dir_from_script, is_mps_dir
+
+
+class AppsTest(TestCase):
+    """Test apps.py module"""
+
+    app_path = join(expanduser('~'), 'projector/apps/app_path')
+
+    def test_get_app_path(self):
+        """The get_app_path method must return correct full path for the specified app path"""
+        user = expanduser('~')
+        self.assertEqual(get_app_path("app_path"), f'{user}/.projector/apps/app_path')
+
+    def test_is_path_to_app_false(self):
+        """The is_path_to_app method must return false if the specified app doesn't exist"""
+        self.assertFalse(is_path_to_app(self.app_path))
+
+    # The parse_version method gives an error when version is a number, for example, 2020124
```

Comment:
For now we consider all versions strings except DDDD.DD{.DD}* as invalid. However, throwing IndexError exception is misleading. So, thanks for catching this, we should emit something more informative.

Reply:
Am I getting it right that if this function gets version = '2020124' the program will crash?
<img width="833" alt="Screen Shot 2021-07-27 at 7 38 13 PM" src="https://user-images.githubusercontent.com/50011154/127193356-06547bb8-b131-4177-8914-be09eda9fc46.png">

Reply:
Yes, application will crash, and we will get exception in the same place where the error occurs. The IDE version is not something, specified by user input. It read from product-info.json file, which has predefined structure (including version format). If this format changes we will not be able to process application correctly, and we should not even try this. However as I already pointed, throwing IndexError is not a good solution. It will be better to create specific exception (like VersionFormatError) and emit it.

Reply:
Thanks! I finally got it